### ISSUE TITLE
ESM format is not available since version 13

### ISSUE THREAD

**Issue by sd-git** [streamich/json-joy#626]
After upgrading from version 11 I noticed missing directories (es6, es2020, esm) from the package distibution.

According to the code this was changed in the build of json-joy since version 13, what was the reason?

We were using json-joy 'esm' format along with Angular. After uprading to version 16 of json-joy, now we have to use '/lib', which is commonJS and Angular build gives complaints about potential optimization issues with 'commonJS library'.

Is there a way to fix this or any plans to provide esm as an option for json-joy package?

Thank you!

**Comment by streamich** [comment]
Hey! The reason for removing it was to simplify the build process and reduce the size of the downloaded NPM package. The idea was that ESM artifacts were not used much, if at all. But since you are bringing up this issue, it might have been not the case (we resume producing ESM artifacts in the future).

**Comment by sd-git** [comment]
Thank you!

**Comment by iyefrat** [comment]
Hey, I would like to second the request to add back the `esm` artifacts to the published package

**Comment by macmillen** [comment]
> Hey! The reason for removing it was to simplify the build process and reduce the size of the downloaded NPM package. The idea was that ESM artifacts were not used much, if at all. But since you are bringing up this issue, it might have been not the case (we resume producing ESM artifacts in the future).

I thought ESM is the modern format of the future? Isn't every library migrating to a ESM-only build?

**Comment by superchris** [comment]
Yes, pretty please support ecmascript modules again 🙏 . This just broke my library that depends json-joy. It also likely will break anyone using modern browser features such as [import maps](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script/type/importmap).
